docs/_config.yml

```
@@ -0,0 +1,50 @@
+title: "Mapviz"
+description: Modular ROS visualization tool for 2D data
+baseurl: "/mapviz" # the subpath of your site, e.g. /blog
+url: "https://jatowler.github.io" # the base hostname & protocol for your site, e.g. http://example.com
```

```suggestion
url: "https://swri-robotics.github.io" # the base hostname & protocol for your site, e.g. http://example.com
```
Might as well go ahead and make this change now; if you do more work on this, you'll want to create a new branch off of master, anyway.